xarray/core/variable.py

```diff
@@ -2069,6 +2079,174 @@ def _to_numeric(self, offset=None, datetime_unit=None, dtype=float):
         )
         return type(self)(self.dims, numeric_array, self._attrs)
 
+    def _unravel_argminmax(
+        self,
+        argminmax: Hashable,
```

Comment:
`mypy` complains that `getattr` only accepts `str`, not `Hashable`. Since we only use it there:
```suggestion
        argminmax: str,
```